profiles/Medellia-setup.yaml

```diff
@@ -0,0 +1,674 @@
+---
```

Comment:
There are a few things we might want to look at since we're going to present this as a Bard config for new users (like cooldown timers and prep messages).

Don't worry about it as part of this PR, but we'll want to look into it later.